frontend/packages/dev-console/src/components/buildconfig/sections/__tests__/ImagesSection.spec.tsx

```diff
@@ -0,0 +1,400 @@
+import * as React from 'react';
+import { configure, render, waitFor } from '@testing-library/react';
+import { Formik, FormikConfig } from 'formik';
+import { Provider } from 'react-redux';
+import store from '@console/internal/redux';
+import userEvent from '../../__tests__/user-event';
+import ImagesSection, { ImagesSectionFormData } from '../ImagesSection';
+
+/*
+jest.mock('@console/internal/components/utils/firehose', () => ({
+  Firehose: ({ resources, children, ...otherProps }) => {
+    return require.requireActual('react').cloneElement(children, {
+      ...otherProps,
+      reduxIDs: ['project.openshift.io~v1~Project'],
+      resources: {
+        project: {
+          kind: 'Project',
+          loaded: true,
+          loadError: '',
+          data: [
+            { metadata: { name: 'project-a' } },
+          ],
+        },
+      }
+    });
+  },
+}))
+*/
```

Comment:
Why is this commented out?

Reply:
Doesn't worked yet, it is required for the test which is marked with .skip at the moment. I'm currently try to fix this test.

Reply:
Solved this now and clean-up-ed all tests.